setup.py

```diff
@@ -300,7 +300,8 @@ def run(self):
             raise RuntimeError("No build directory. Run `python setup.py build` before trying to run tests.")
         os.environ['GRAMPS_RESOURCES'] = '.'
         all_tests = unittest.TestLoader().discover('.', pattern='*_test.py')
-        unittest.TextTestRunner(verbosity=self.verbose).run(all_tests)
+        unittest.TextTestRunner(
+            verbosity=2 if self.verbose else 0).run(all_tests)
```

Comment:
That's weird; I was running via `python3 -m setup.py test -v` and I only got a row of dots.
`python3 -m setup.py test -q` made it go quiet.  So I thought that the `-v` was having no extra effect, reinforced by the distutils docs.
I now see that `python3 -m setup.py -v test` with '-v' before the 'test' has the desired effect.  The '-q' seems to work in either position...  I'll remove that part of the PR.